Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps.tutorial

```diff
@@ -90,15 +112,42 @@
     }
     
     @Steps {
+      Before anything, we need to fix a few compiler errors. We'll start with the initialization
+      of the list of `SyncUp` in the `deletion()` test.
+
+      When using a `Shared` state with a persistence strategie like we did in the `SyncUpList` feature, 
+      we can initialize it from outside.
+
+      @Step {
+        Go to `SyncUpsListTests.swift` and move the initialization of the syncUp we want to delete outside of 
+        the `TestStore` initialization.
+
+        > note: the `fileStorage` persistence strategy uses an internal dependency for changing how files 
+        > are written to the disk and loaded from disk. In tests the dependency will forgo any interaction 
+        > with the file system and instead write data to a `[URL: Data]` dictionary, and load data from that 
+        > dictionary. That emulates how the file system works, but without persisting any data to the global 
+        > file system.
```

Comment:
📓 In the tests, I explain how to init the shared property from the outside of the TestStore in the deletion test and add a note to also explain how the persistence strategy work in tests